lib/purescript-cst/package.yaml

```diff
@@ -0,0 +1,77 @@
+name: purescript-cst
+version: '0.13.6'
+synopsis: PureScript Programming Language Concrete Syntax Tree
+description: The surface syntax of the PureScript Programming Language.
```

Comment:
Also not sure what to put here. Most of the wording is from [the PR to add the CST](https://github.com/purescript/purescript/pull/3608). Down to change it to whatever.